functionnal_tests/cypress/integration/login/login_after_disconnection.js

```diff
@@ -0,0 +1,72 @@
+import { SELECTORS as s } from '../../support/generic_selector_commands'
+import { PAGES as p } from '../../support/urls_commands'
+
+describe('Login after a disconnection ', function () {
+    const login = 'admin@admin.admin'
+    const username = 'test'
+    const email = 'test@test.test'
+    const pwd = 'testPwd'
+
+    before(() => {
+        cy.resetDB()
+        cy.setupBaseDB()
+        cy.visitPage({pageName: p.LOGIN, params: {loginParam: '?dc=1'}})
+
+        cy.getTag({selectorName: s.LOGIN_PAGE_CARD})
+            .find('input[type=email]')
+            .type(login)
+
+        cy.getTag({selectorName: s.LOGIN_PAGE_CARD})
+            .find('input[type=password]')
+            .type(login)
+
+        cy.getTag({selectorName: s.LOGIN_PAGE_CARD})
+            .find('.loginpage__card__form__btnsubmit')
+            .click()
+    })
+
+    it('should able to add user', function () {
+        cy.wait(2000)
+        cy.getTag({selectorName: s.HEADER})
+            .find('.adminlink__btn.btn')
+            .click()
+
+        cy.getTag({selectorName: s.HEADER})
+            .find('a[href="/ui/admin/user"]')
```

Comment:
this selector is too fragile, you should add a data-cy to .jsx file that creates it